certification/internal/utils/utils.go

```diff
@@ -0,0 +1 @@
+package utils
```

Comment:
I believe Golang best practices recommend avoiding packages named util as the package could be a catch all for lots of unrelated things. Is there a more descriptive package name we could use?

Reply:
That makes sense. How does `preflightutils` sound?

Reply:
Maybe just remove *utils.go` so we're not defining a utils package? Then you keep the import path? Would that work @tonytcampbell?

The blurb on package names, for reference:

> Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains. This makes it harder for clients to use the package and makes it harder for maintainers to keep the package focused. Over time, they accumulate dependencies that can make compilation significantly and unnecessarily slower, especially in large programs. And since such package names are generic, they are more likely to collide with other packages imported by client code, forcing clients to invent names to distinguish them.
Source: https://blog.golang.org/package-names

Reply:
Thanks for the blurb @komish.  That explains it well.  @samira-barouti can we just get rid of the utils package altogether?  I think what @komish recommends above would work.

Reply:
I was planning to use this utils package as a parent package which contains multiple sub packages. Right now, we have `file` as a sub-package; I wanted to migrate the `util.go` in the shell package, and create a `container` sub-package. 
I do not have any strong opinion on keeping this; my intention was to introduce a sort of structure in this directory, and enable the future extension of this package to include other utility functions as subpackages. If the consensus is to remove this file, I am good with it :)

Reply:
@samira-barouti You can leave the directory structure as is even without utils.go file. You would keep the organizational structure that way, but we would have no importable `utils` package. 

You'd still see something like `import .../internal/utils/file` for your import statements. 

Does that make sense?

Reply:
Beautiful! Thank you @komish. That makes sense